editoast/editoast_schemas/src/train_schedule/path_item.rs

```diff
@@ -32,6 +34,33 @@ pub struct PathItem {
 #[serde(untagged, deny_unknown_fields)]
 pub enum PathItemLocation {
     TrackOffset(#[schema(inline)] TrackOffset),
+    OperationalPointReference(#[schema(inline)] OperationalPointReference),
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema, Hash)]
+pub struct OperationalPointReference {
+    #[serde(flatten)]
+    #[schema(inline)]
+    pub reference: OperationalPointIdentifier,
+    #[serde(default)]
+    pub track_reference: Option<TrackReference>,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema, Hash)]
+#[serde(untagged, deny_unknown_fields)]
+pub enum TrackReference {
+    Id {
+        #[schema(inline)]
+        track_id: Identifier,
+    },
+    Label {
+        track_label: NonBlankString,
```

Comment:
Any reason why `track_id` is `#[schema(inline)]` and `track_label` is not?

Reply:
Sadly, it's impossible to inline `Identifier`. The only way to inline it, is to put a field `track_id: Identifier` with `schema(inline)`

Reply:
Dosen't that work?

```rust
#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema, Hash)]
#[serde(deny_unknown_fields)]
pub enum TrackReference {
    TrackId(#[schema(inline)] Identifier),
    TrackLabel(NonBlankString),
}
```